modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/PartitionHashRecordV2.java

```diff
@@ -19,7 +19,6 @@
 import java.io.IOException;
 import java.io.ObjectInput;
 import java.io.ObjectOutput;
-
```

Comment:
Nothing changed in this file except this line

Reply:
Fixed.